### DISCUSSION TITLE
Extra long timelapse stops after a few dozen pictures and camera stays on

### DISCUSSION THREAD

**Discussion by The-NS** [gopro/labs#468] [Q&A]
Hello, I recently re-installed GoPro Labs on my Hero 8 Black and the Extra Long Timelapse is not working correct.  It starts just fine (powers on, takes a pic, powers off), but after a while it stays on and stops taking pictures.  The program has not ended though; if I do a manual restart it picks right back up.  I previously had labs installed on this computer and used this feature just fine.  How do I troubleshoot this?

Thanks!!

**Comment by dnewman-gpsw** [comment]
look in MISC/qrlog.txt is should help find what went wrong.
